docs/labs/4-ACM.md

```diff
@@ -325,6 +325,17 @@ we would notice that we would have a picture like this.
 
 ![Picture with GOval, GLine, GRect, GLabel](lab4media/media/image4.png)
 
+There are a lot of other methods that ```GOval```, ```GLine```, ```GRect```, and ```GLabel```
```

Comment:
Can you use single backticks here instead of triple backticks?
Also would be nice if the line breakup was done semantically - each sentence is on its own line and for those that are very long, you break either at a comma, or at some preposition that indicates a long clause (like "that")

Reply:
I think the rest of the docs are using triple backticks and that is frankly incorrect when it is sprinkled within code, it should be single backticks there, and triple backticks (`) should be used for code blocks